app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -0,0 +1,154 @@
+        <p class="bold">
+          I promise the level of satisfaction is like nothing you’ll get in another job. I wake up every morning and want to get into school.
+        </p>
+      </blockquote>
+
+      <figcaption>Phil, teacher</figcaption>
+    </figure>
+  </section>
+
+  <section class="columns">
+    <div>
+      <h2 class="heading-s">What explore teaching advisers can do</h2>
+
+      <p>
+        Your adviser can help you:
+      </p>
+    </div>
+
+    <div>
+      <ul class="badges">
+        <li class="badge">understand what teaching is really like</li>
+        <li class="badge">connect with other students</li>
+        <li class="badge">build your confidence and transferable skills</li>
+        <li class="badge">get classroom experience</li>
+        <li class="badge">understand the process to become a teacher</li>
+      </ul>
+    </div>
+  </section>
+
+  <section>
+    <h2 class="heading-s">How to get an explore teaching adviser</h2>
+
+    <p>
+        To be eligible for an explore teaching adviser, you just need:
+    </p>
+
+    <ul>
+      <li>
+        to be studying for an undergraduate degree, not in your final year of study
+      </li>
+      <li>
+        an interest in teaching
+      </li>
+    </ul>
+
+    <p>
+    If you're in your final year of study or you already have a degree, you can still get a teacher training adviser.
+    </p>
+
+    <p>
+      <a href="/teacher-training-advisers">Find out how a teacher training adviser could help you</span></a>.
+    </p>
+
+    <p class="call-to-action__action">
+      <a href="https://adviser-getintoteaching.education.gov.uk/?channel=222750047">Get an <span>explore teaching adviser</span></a>
+    </p>
+  </section>
+
+  <%= render SubjectSpecific::TextBlockWithOneImageComponent.new(
+    heading: "Is teaching right for me?",
+    colour: "yellow",
+    image_path: "media/images/content/hero-images/0011.jpg",
+    image_alt: "Child pointing something out to a teacher in the classroom.",
+  ) %>
+
+  <section>
+    <h2 class="heading-s">Use your subject passion</h2>
+
+    <p>
+    Bring your subject to life for your pupils. Help them be inspired by it just like you.
+    </p>
+    <p>
+      <a href="/blog/the-head-of-science-inspiring-students-in-blackpool">Find out how Jessica brings real life science stories into the classroom.</a>.
+    </p>
+  </section>
+
+    <section>
+    <h2 class="heading-s">Join a community</h2>
+
+    <p>
+    Teaching is about more than just what happens in the classroom. You become part of a tight-knit team with your fellow teachers and support staff.
+    </p>
+        <p>
+      <a href="/blog/the-fellowship-of-teachers">Explore the friendship Mike and Phil have made outside of the classroom</a>.
+    </p>
+  </section>
+
+  <section>
+    <h2 class="heading-s">Make a difference</h2>
+
+    <p>
+      Every lesson shapes a life. Teachers make a significant impact on a child's ability, confidence, and determination to succeed.
```

Comment:
The other two sections are a bit more direct ie 'you' and 'your' - could this one be as well?

eg 'As a teacher, you can make...'?